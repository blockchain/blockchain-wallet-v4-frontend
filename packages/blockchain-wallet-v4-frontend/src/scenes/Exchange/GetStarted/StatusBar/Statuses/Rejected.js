import React from 'react'
import styled from 'styled-components'
import { FormattedMessage } from 'react-intl'

import { Text } from 'blockchain-info-components'

const Wrapper = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: flex-start;
  align-items: flex-start;
  width: 100%;
  padding: 16px;
  box-sizing: border-box;
  background-color: #f26c57;
  border-radius: 4px;

  & > :not(:first-child) {
    margin-top: 8px;
  }

  @media (min-width: 960px) {
    width: 500px;
  }
`

export const Rejected = () => (
  <Wrapper>
    <Text color='white' size='14px' weight={400}>
      <FormattedMessage
        id='scenes.exchange.getstarted.status.rejected.title'
        defaultMessage='Account verification failed'
      />
    </Text>
    <Text size='14px' weight={300}>
      <FormattedMessage
        id='scenes.exchange.getstarted.status.rejected.description'
        defaultMessage='Unfortunately we had some trouble with the documents that you’ve supplied and we can’t verify your account at this time. '
      />
    </Text>
  </Wrapper>
)

export default Rejected
