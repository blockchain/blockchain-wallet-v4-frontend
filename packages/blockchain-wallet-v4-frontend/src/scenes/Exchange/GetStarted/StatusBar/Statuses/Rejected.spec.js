import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import { Rejected } from './Rejected'

jest.mock('blockchain-info-components', () => ({
  Button: 'button',
  ICon: 'icon'
}))

describe('Rejected', () => {
  it('renders correctly', () => {
    const component = shallow(<Rejected />)
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  })
})
