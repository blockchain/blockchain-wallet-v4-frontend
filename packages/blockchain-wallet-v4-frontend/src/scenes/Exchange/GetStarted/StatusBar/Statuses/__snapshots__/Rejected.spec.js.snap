// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rejected renders correctly 1`] = `
<Rejected__Wrapper>
  <Component
    color="white"
    size="14px"
    weight={400}
  >
    <FormattedMessage
      defaultMessage="Account verification failed"
      id="scenes.exchange.getstarted.status.rejected.title"
      values={Object {}}
    />
  </Component>
  <Component
    size="14px"
    weight={300}
  >
    <FormattedMessage
      defaultMessage="Unfortunately we had some trouble with the documents that you’ve supplied and we can’t verify your account at this time. "
      id="scenes.exchange.getstarted.status.rejected.description"
      values={Object {}}
    />
  </Component>
</Rejected__Wrapper>
`;
